src/rdo.rs

```diff
@@ -439,9 +439,14 @@ pub fn compute_distortion_scale<T: Pixel>(
   let x = frame_bo.0.x >> IMPORTANCE_BLOCK_TO_BLOCK_SHIFT;
   let y = frame_bo.0.y >> IMPORTANCE_BLOCK_TO_BLOCK_SHIFT;
 
```

Comment:
The `compute_distortion_scale` method should be renamed to just `distortion_scale` since it doesn't really compute anything anymore.

Reply:
I'll bake anothe patch on top of it. I wanted to keep the changes to the bare minimum.